vmdb/app/controllers/ops_controller/db.rb

```diff
@@ -230,11 +230,8 @@ def db_get_info(nodetype)
   end
 
   def get_indexes(tb)
-    indexes = Array.new
-    tb.vmdb_indexes.sort{|a,b| a.name <=> b.name}.each do |idx|
-      indexes.push(idx) if idx.vmdb_table.type == "VmdbTableEvm"
-    end
-    return indexes
+    return [] unless tb.kind_of?(VmdbTableEvm)
+    tb.vmdb_indexes.sort_by(&:name)
```

Comment:
idx.vmdb_table is the same object as tb, so it' can't ever be wrong, because of the guard clause.